src/instructlab/cli/model/train.py

```diff
@@ -397,10 +397,12 @@ def train(
     phased_phase1_data: pathlib.Path | None,
     phased_phase1_num_epochs: int | None,
     phased_phase1_samples_per_save: int | None,
+    phased_phase1_learning_rate: float | None,
```

Comment:
might need to add a click flag for this as well that should read the default from the config file

Reply:
adding now

Reply:
added